.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Ask a question
+    url: https://stackoverflow.com/questions/tagged/amazon-sagemaker
```

Comment:
should we point this to the [MXNet forums](https://discuss.mxnet.io/) instead?

Reply:
Looking at previous question issues, and it seems like they've been mostly SageMaker-specific. I could see either being appropriate, though.

Reply:
Actually, it wouldn't hurt to add another contact link for the MXNet forums. I'll go ahead and do that.